spk/flexget/Makefile

```diff
@@ -1,35 +1,36 @@
 SPK_NAME = flexget
-SPK_VERS = 2.9.14
-SPK_REV = 6
-SPK_ICON = src/flexget.png
+SPK_VERS = 3.1.79
+SPK_REV = 7
+SPK_ICON = src/${SPK_NAME}.png
 
+BUILD_DEPENDS = cross/python3 cross/setuptools cross/pip cross/wheel
 WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>=2.7.8-11"
+SPK_DEPENDS = "python3>=3.7.7"
 
-MAINTAINER = Diaoul
+MAINTAINER = SynoCommunity
 DESCRIPTION = FlexGet is a multipurpose automation tool for content like torrents, nzbs, podcasts, comics, series, movies, etc. It can use different kinds of sources like RSS-feeds, html pages, csv files, search engines and there are even plugins for sites that do not provide any kind of useful feeds.
 DISPLAY_NAME = FlexGet
-BETA = 1
-CHANGELOG = "Update Flexget to version 2.9.14"
+CHANGELOG = "Update FlexGet to version 3.1.79. Please note that FlexGet now runs on Python 3. Make sure your post/pre-processing scripts work with Python 3!"
```

Comment:
This notice should be in install/upgrade wizard, not in change log, in my opinion.

Reply:
Sorry, but disagree with you. Let me explain why.
If I have these sentences in the changelog, I will see them at https://synocommunity.com/package/flexget.
And if I try to install the package, I see a screen with requirements to install Python3 as a dependency.
So I think everything is in place.

Reply:
Or I can just remove it from the changelog.

Reply:
@ymartin59 updated the CHANGELOG.